keyboards/eek/keymaps/ledtest/rules.mk

```diff
@@ -0,0 +1,2 @@
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+#RGB_MATRIX_ENABLE = WS2812       # Enable for pretty RGB matrix effects
```

Comment:
```suggestion
#RGB_MATRIX_ENABLE = yes  # Enable for pretty RGB matrix effects
```

Reply:
@fauxpark if I make this change and the one suggested for eek/rules.mk, I receive this error:

```
Compiling: quantum/rgb_matrix_drivers.c                                                            quantum/rgb_matrix_drivers.c: In function 'init':
quantum/rgb_matrix_drivers.c:33:21: error: 'DRIVER_ADDR_1' undeclared (first use in this function); did you mean 'TIMER_AVR_H'?
     IS31FL3731_init(DRIVER_ADDR_1);
                     ^~~~~~~~~~~~~
                     TIMER_AVR_H
quantum/rgb_matrix_drivers.c:33:21: note: each undeclared identifier is reported only once for each function it appears in
quantum/rgb_matrix_drivers.c: In function 'flush':
quantum/rgb_matrix_drivers.c:87:35: error: 'DRIVER_ADDR_1' undeclared (first use in this function); did you mean 'TIMER_AVR_H'?
     IS31FL3731_update_pwm_buffers(DRIVER_ADDR_1, 0);
                                   ^~~~~~~~~~~~~
                                   TIMER_AVR_H
 [ERRORS]
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:386: .build/obj_eek_silk_down_ledtest/quantum/rgb_matrix_drivers.o] Error 1
```

Reply:
Make sure that your repo is up to date. You may need to rebase it.

Reply:
Dang! I’ll do it tonight! I swear I just did that last week, lol

Reply:
Oi, that was definitely my problem.  Would you guys please check it again?